guacamole/src/main/java/org/glyptodon/guacamole/net/basic/GuacamoleSession.java

```diff
@@ -53,12 +53,12 @@
     /**
      * The credentials provided when the user logged in.
```

Comment:
I thought so, but I'm too close to this to have objective opinions. Think we should change it to "... when the user authenticated"?